packages/components/src/internal/components/buttons/DisableableButton.tsx

```diff
@@ -22,14 +23,14 @@ export const DisableableButton: FC<Props> = memo(props => {
                         </Popover>
                     }
                 >
-                    <div className="disabled-button-with-tooltip">
+                    <div className={'disabled-button-with-tooltip ' + className}>
```

Comment:
If the className is undefined the div will look like this in the DOM:

`<div class="disabled-button-with-tooltip undefined">`

Which we don't really want. You can set a default value above though where we desctructure the values from props:

```
    const { bsStyle, className = '', disabledMsg, title, onClick, children } = props;
```